quirks/shared-credentials.json

```diff
@@ -234,6 +234,16 @@
         ],
         "fromDomainsAreObsoleted": true
     },
+    {
+        "from": [
+            "gazduire.com.ro",
+            "gazduire.net"
+        ],
+        "to": [
+            "admin.ro"
```

Comment:
Oh, but both redirect to admin.ro when it comes time to sign in!

Reply:
Yeah, I explained it in the description, but buried after the last checklist item 🙈